app/src/main/res/values-pt-rBR/strings.xml

```diff
@@ -219,10 +219,10 @@
     <string name="action_msg_copy">Copiar</string>
     <string name="action_msg_quote">Citar</string>
     <string name="action_msg_delete">Remover</string>
-    <string name="action_msg_pin">Pinar mensagem</string>
-    <string name="action_msg_unpin">Despinar mensagem</string>
-    <string name="action_msg_star">Favoritar mensagem</string>
-    <string name="action_msg_unstar">Desfavoritar messagem</string>
+    <string name="action_msg_pin">Afixar mensagem</string>
+    <string name="action_msg_unpin">Desafixar mensagem</string>
```

Comment:
`Desfixar` following the same idea above.

Reply:
https://github.com/RocketChat/Rocket.Chat.Android/pull/1941/commits/7259fe61092a0102ccf0c071e638b6b5053fb597